x-pack/filebeat/module/azure/platformlogs/test/platformlogs.log-expected.json

```diff
@@ -0,0 +1,46 @@
+[
+    {
+        "cloud" : {
+            "region" : "West Europe",
+            "provider" : "azure"
+        },
+        "@timestamp" : "2020-11-03T09:06:42.000Z",
+        "event" : {
+            "action" : "Retreive ConsumerGroup",
+            "ingested" : "2020-11-05T12:51:52.507448200Z",
+            "kind" : "event",
+            "outcome" : "succeeded"
+        },
+        "azure" : {
+            "subscription_id" : "7657426D-C4C3-44AC-88A2-3B2CD59E6DBA",
+            "resource" : {
+                "name" : "OBSTESTEVENTHUBS",
+                "id" : "/SUBSCRIPTIONS/7657426D-C4C3-44AC-88A2-3B2CD59E6DBA/RESOURCEGROUPS/OBS-TEST/PROVIDERS/MICROSOFT.EVENTHUB/NAMESPACES/OBSTESTEVENTHUBS",
+                "provider" : "MICROSOFT.EVENTHUB/NAMESPACES",
+                "group" : "OBS-TEST"
+            },
+            "platformlogs" : {
+                "Status" : "Succeeded",
```

Comment:
This field doesn't have a mapping, right? I see you use them to fill other fields, should it be dropped or mapped?

Reply:
added mapping for this field